src/components/widgets/Plotter.vue

```diff
@@ -290,7 +292,7 @@ const renderCanvas = (): void => {
 
```

Comment:
```suggestion
ctx.moveTo(0, canvasHeight / 2)

```
This isn't letting me suggest on the actual line just above this one, but if we're generally tending towards the plotted data being at/near the center line, then the plot line should start in the middle (so it doesn't have to draw a line up the left side from the bottom corner).